bucoffea/plot/studies/wtag_sf/mistagSF.py

```diff
@@ -0,0 +1,108 @@
+from array import array
+from bucoffea.plot.stack_plot import *
+from klepto.archives import dir_archive
+import ROOT
+
+outfilename = "../../../data/sf/ak8/wtag_mistag_SF.root"
+outfile = ROOT.TFile.Open(outfilename,'recreate')
+newbin = hist.Bin('jetpt',r'AK8 jet $p_{T}$ (GeV)', [200,300,400,800])
+
+inpath = "../../input/merged"
+region = 'cr_2m_1ak8_inclusive_v'
+
+plot_dir = 'output/'
+if not os.path.exists(plot_dir):
+    os.makedirs(plot_dir)
+
+acc = dir_archive(
+    inpath,
+    serialized=True,
+    compression=0,
+    memsize=1e3,
+    )
+acc.load('sumw')
+acc.load('sumw_pileup')
+acc.load('nevents')
+
+def divide_sumw2(sumw_a, sumw2_a, sumw_b, sumw2_b): #return (sumw_c, sumw2_c) for c=a/b
+    # check that the inputs have the same shape
+    assert len(set((sumw_a.shape, sumw_b.shape, sumw2_a.shape, sumw2_b.shape))) == 1
+    sumw_c  = np.zeros(sumw_a.shape)
+    sumw2_c = np.zeros(sumw2_a.shape)
+    for ientry in range(len(sumw2_c)):
+        if sumw2_b[ientry]>0:
+            sumw_c[ientry] = sumw_a[ientry] / sumw_b[ientry]
+            sumw2_c[ientry] = (sumw_a[ientry]**2 * sumw2_b[ientry] + sumw_b[ientry]**2 * sumw2_a[ientry]) / (sumw_b[ientry]**2 * sumw_b[ientry]**2)
```

Comment:
@siqiyyyy just noticed this. When you are calculating an efficiency, the numerator and denominator share a large number of their events and are therefore not statistically independent. Can't use Gaussian error propagation in that case, but rather the clopper pearson interval, see e.g. https://github.com/CoffeaTeam/coffea/blob/master/coffea/hist/plot.py#L383